druid-shell/Cargo.toml

```diff
@@ -19,7 +19,7 @@ default-target = "x86_64-pc-windows-msvc"
 # NOTE: When changing the piet or kurbo versions, ensure that
 #       the kurbo version included in piet is compatible with the kurbo version specified here.
 piet-common = "0.1.1"
-kurbo = "0.6.0"
+kurbo = "0.6.1"
```

Comment:
Do we need to directly import kurbo still, or can we use `piet::kurbo`?

Reply:
We don't need to, no. I think the thinking was that keeping the import here will allow for easier bumps if druid wants some new feature, without having to update piet.

Reply:
this is a way to bump our minimum kurbo version without having to do a piet release.